featuretools/feature_discovery/convertors.py

```diff
@@ -0,0 +1,178 @@
+from __future__ import annotations
+
+from typing import Dict, List
+
+import pandas as pd
+from woodwork.logical_types import LogicalType
+
+from featuretools.feature_base.feature_base import (
+    FeatureBase,
+    IdentityFeature,
+    TransformFeature,
+)
+from featuretools.feature_discovery.LiteFeature import LiteFeature
+from featuretools.primitives import TransformPrimitive
+from featuretools.primitives.base.primitive_base import PrimitiveBase
+
+FeatureCache = Dict[str, FeatureBase]
+
+
+def convert_featurebase_to_feature(feature: FeatureBase) -> LiteFeature:
+    """
+    Convert a FeatureBase object to a LiteFeature object
+
+    Args:
+        feature (FeatureBase):
+
+    Returns:
+       LiteFeature - converted LiteFeature object
+    """
+    base_features = [convert_featurebase_to_feature(x) for x in feature.base_features]
+
+    name = feature.get_name()
+    col_schema = feature.column_schema
+
+    logical_type = col_schema.logical_type
+    if logical_type is not None:
+        assert issubclass(type(logical_type), LogicalType)
+        logical_type = type(logical_type)
+
+    tags = col_schema.semantic_tags
+
+    if isinstance(feature, IdentityFeature):
+        primitive = None
+    else:
+        primitive = feature.primitive
+        assert isinstance(primitive, PrimitiveBase)
+
+    return LiteFeature(
+        name=name,
+        logical_type=logical_type,
+        tags=tags,
+        primitive=primitive,
+        base_features=base_features,
+        # TODO: replace this with dataframe name?
+        df_id=None,
+    )
+
+
+def to_transform_feature(
```

Comment:
Nit: just to be consistent, I think we should name this with the `convert x to y` format used in this PR